pkg/volume/volume.go

```diff
@@ -121,6 +121,17 @@ type Deleter interface {
 	Delete() error
 }
 
+// Attacher can attach a volume to a node.
+type Attacher interface {
+	Volume
+	Attach() error
+}
+
+// Detacher can detach a volume from a node.
+type Detacher interface {
```

Comment:
Probably?  I'm not sure either really needs to inline `Volume`.  I haven't digested that thought fully.

Does any other volume interface that's not Builder/Cleaner (soon to be Mounter/Unmounter) need `Volume`?